src/styles/base/_override.links.scss

```diff
@@ -12,6 +12,7 @@ $link-inverse-hover-color: $color-muted-inverse;
 $link-inverse-active-color: $color-muted-inverse;
 
 // Link underline
+$text-decoration-color: $color-primary-alt;
```

Comment:
Can we rename this to `$link-underline-color` ? Also It would be great to add this variable to the core as well to make it available for HCgov as well.

Reply:
Thank you for noticing this. This can be taken out, I think it was left in while I was doing some testing but it doesn't apply to any styles.